libres/lib/include/ert/analysis/ies/ies_config.hpp

```diff
@@ -22,7 +22,6 @@
 #include <variant>
 
 namespace ies {
-namespace config {
 
 constexpr double DEFAULT_TRUNCATION = 0.98;
```

Comment:
Must admit that I like that these are under the `config` namespace (but not `Config`, that is a bit redundant :sweat_smile: )